src/http/server_context.rs

```diff
@@ -4,23 +4,70 @@ use async_graphql::dataloader::{DataLoader, NoCache};
 use async_graphql::dynamic;
 use derive_setters::Setters;
 
-use crate::blueprint::Blueprint;
+use crate::blueprint::{Blueprint, Definition};
 use crate::config::Server;
+use crate::directive::DirectiveCodec;
+use crate::group_by::GroupBy;
 use crate::http::{DefaultHttpClient, HttpDataLoader};
+use crate::lambda::{Expression, Operation};
 
 #[derive(Setters, Clone)]
 pub struct ServerContext {
   pub schema: dynamic::Schema,
   pub http_client: DefaultHttpClient,
   pub server: Server,
-  pub data_loader: Arc<DataLoader<HttpDataLoader<DefaultHttpClient>, NoCache>>,
+  pub data_loaders: Vec<Arc<DataLoader<HttpDataLoader<DefaultHttpClient>, NoCache>>>,
+}
+
+fn assign_id(blueprint: &mut Blueprint) -> &Blueprint {
+  for (index, def) in blueprint.definitions.iter_mut().enumerate() {
+    if let Definition::ObjectTypeDefinition(def) = def {
+      for field in &mut def.fields {
+        if let Some(Expression::Unsafe(Operation::Endpoint(req_template))) = &mut field.resolver {
+          req_template.id = Some(index);
+        }
+      }
+    }
+  }
+  blueprint
+}
+
+pub fn get_data_loaders(
+  blueprint: &Blueprint,
+  server: Server,
+) -> Vec<Arc<DataLoader<HttpDataLoader<DefaultHttpClient>, NoCache>>> {
+  let mut data_loaders = Vec::new();
+  for def in blueprint.definitions.iter() {
+    if let Definition::ObjectTypeDefinition(def) = def {
+      for field in &def.fields {
+        if let Some(Expression::Unsafe(Operation::Endpoint(_))) = &field.resolver {
+          let mut data_loader = Arc::new(
+            HttpDataLoader::new(DefaultHttpClient::default(), None)
```

Comment:
FIX: client should be passed and not created.

Reply:
Can enrich the endpoint expression with the DL directly creating a new Blueprint.